### PR TITLE
Add apollo-link-state to frontend

### PR DESCRIPTION
**Pull request by jaimemendozadev**
Setup cache for local state management

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -6,12 +6,28 @@ import { ApolloClient } from 'apollo-client';
...
+const stateLink = withClientState({
+  cache,
+  defaults: defaultState,
+});
+
 
 const client = new ApolloClient({
-  link: new HttpLink({ uri: 'http://localhost:3050/graphql' }),
-  cache: new InMemoryCache()
+  link: ApolloLink.from([stateLink, new HttpLink({ uri: 'http://localhost:3050/graphql' })]),
```
Comment (nooysters) [inline client/src/App.jsx]: consider using an environment variable for your host and protocol:
```

File: client/src/components/DrinkList.jsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { graphql } from 'react-apollo';
 import gql from 'graphql-tag';
 import * as queries from '../graphql/queries';
 
-const {getAllDrinks} = queries;
+const {getAllDrinks, getCurrentEmployee} = queries;
 
 class DrinkList extends Component {
```
Comment (nooysters) [inline client/src/components/DrinkList.jsx]: Unless you are using lifecycle hooks use pure functional components.

File: client/src/graphql/queries.js [hunk 3]
```diff
@@ -9,4 +9,13 @@ export const getAllDrinks = gql`
...
+`;
+
+
+export const getCurrentEmployee = gql`
+  query getCurrentEmployee {
+    currentEmployee @client {
+      name
+    }
+  } 
 `;
```
Comment (nooysters) [inline client/src/graphql/queries.js]: your linter should make you have an empty line at the end of your file. You can use husky to run git pre-commit hooks to lint the code. https://github.com/typicode/husky

**Comment by jaimemendozadev** [comment]
@nooysters Made changes per the linter instructions and changed the `DrinkList` component

**Pull request merged**